packages/flutter_chat/lib/src/screens/chat_profile_screen.dart

```diff
@@ -139,57 +142,60 @@ class _Body extends StatelessWidget {
 
     var chatUserDisplay = Wrap(
       children: [
-        ...chat!.users.map(
-          (tappedUser) => Padding(
-            padding: const EdgeInsets.only(
-              bottom: 8,
-              right: 8,
-            ),
-            child: InkWell(
-              onTap: () {
-                onTapUser?.call(tappedUser);
-              },
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.start,
-                mainAxisSize: MainAxisSize.min,
-                children: [
-                  FutureBuilder<UserModel>(
-                    future: service.getUser(userId: tappedUser).first,
-                    builder: (context, snapshot) {
-                      if (snapshot.connectionState == ConnectionState.waiting) {
-                        return const CircularProgressIndicator();
-                      }
+        if (chat != null) ...[
```

Comment:
I know this is the only change here, but this screen really needs to be changed in terms of readability.

Reply:
I did a little refactor (removing duplicate code but I will extract widgets at some later point)